app/models/sponsor.rb

```diff
@@ -2,6 +2,8 @@ class Sponsor < ActiveRecord::Base
   include Initializer
   NEW_CITY_MENU_OPTION = '**Add New**'
   PAYMENT_PLANS = ['Monthly', 'Every Two Months', 'Every Four Months', 'Every Six Months', 'Annually', 'Other']
+  PRIORITY_COUNTRIES= %w(SA TR AE GB)
+  EXCLUDED_COUNTRYS= %w(IL BQ BV IO KY CX CC CW FK TF GG HM IM JE YT AN NF NO PN BL SH MF PM SX GS SS SJ TK TC UM AX)
```

Comment:
not sure why "NO" : "Norway"  is not translating.
`I18n.t 'NO'` => `"translation missing: ar.NO"`
and we have NO defined in ar.yml